fuel/app/classes/presenter/welcome/404.php

```diff
@@ -16,6 +16,27 @@ class Presenter_Welcome_404 extends Presenter
 	 */
 	public function view()
 	{
+		$fufufu = "jjkpowek;rlwke;lk;lk;lk";
+		$fddufufu = "jjkpowek;rlwke;lk;lk;lk";
+		$fufddufu = "jjkpowek;rlwke;lk;lk;lk";
+		$fufufddddu = "jjkpowek;rlwke;lk;lk;lk";
+
+	$fufufdddddddu = "jjkpowek;rlwke;lk;lk;lk";
+
+	$fdasddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddufufdddsadsasdu = "jjkpowek;rlwke;lk;lk;lk";
+    $djasijoi = 12312412421312312;
+
+		$fdddufu123fddddu = "jjkpowek;rlwke;lk";
+
+$a = 1;
```

Comment:
__[development][phpmd]__
* Avoid variables with short names like $a. Configured minimum length is 3.
* Avoid unused local variables such as '$a'.